src/main/java/nl/nn/testtool/web/ApiAuthorizationFilter.java

```diff
@@ -89,6 +89,7 @@ public void setObserverRoles(List<String> observerRoles) {
 		if (constructorDone) log.info("Set observer roles");
 		addConfigurationPart("GET/"  + ApiServlet.LADYBUG_API_PATH + "/testtool.*$", observerRoles);
 		addConfigurationPart("POST/" + ApiServlet.LADYBUG_API_PATH + "/testtool/transformation[/]?$", observerRoles); // [/]? because frontend is currently not using a slash at the end
+		addConfigurationPart("PUT/"  + ApiServlet.LADYBUG_API_PATH + "/testtool/views/.*$", observerRoles);
```

Comment:
Can you make it specific instead of wildcard? Observer role should have the least possible authorizations and the clearest human readable list of authorizations (especially for PUT)